clients/runtime/src/rpc.rs

```diff
@@ -456,6 +470,9 @@ pub trait UtilFuncs {
 	/// Gets the current height of the parachain
 	async fn get_current_chain_height(&self) -> Result<u32, Error>;
 
+	/// Gets the ID of the native currency.
+	fn get_native_currency_id(&self) -> CurrencyId;
```

Comment:
Wouldn't we also need a utility function for the relay chain currency ID?

Reply:
I checked interlays code and they also don't use it. Fine if we don't add it then I guess.